Default.php

```diff
@@ -8,38 +8,51 @@
 
 require_once __DIR__ . '/ContextLoader.php';
 
-if (isset($_ENV['DDEV_DATABASE_FAMILY'])) {
-    $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections']['Default']['dbname'] = 'db';
-    $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections']['Default']['host'] = 'db';
-    $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections']['Default']['user'] = 'db';
-    $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections']['Default']['password'] = 'db';
-}
-if (isset($_ENV['MYSQL_DB'])) {
-    $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections']['Default']['dbname'] = $_ENV['MYSQL_DB'];
-}
-if (isset($_ENV['MYSQL_HOST'])) {
-    $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections']['Default']['host'] = $_ENV['MYSQL_HOST'];
-}
-if (isset($_ENV['MYSQL_PORT'])) {
-    $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections']['Default']['port'] = $_ENV['MYSQL_PORT'];
-}
-if (isset($_ENV['MYSQL_USER'])) {
-    $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections']['Default']['user'] = $_ENV['MYSQL_USER'];
-}
-if (isset($_ENV['MYSQL_PASS'])) {
-    $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections']['Default']['password'] = $_ENV['MYSQL_PASS'];
-}
+call_user_func(function() {
+    if (isset($_ENV['DDEV_DATABASE_FAMILY'])) {
```

Comment:
@baschny, vielleicht in dem Zug auch gleich die "richtige" ENV var nutzen? 

https://github.com/ddev/ddev/blob/master/pkg/ddevapp/typo3/AdditionalConfiguration.php#L9